.travis.yml

```diff
@@ -22,12 +23,12 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - npm config set spin false
-  - npm install -g bower
-  - npm install phantomjs-prebuilt
+  - npm install --no-spin -g yarn
+  - yarn global add bower
+  - yarn global add phantomjs-prebuilt
 
 install:
-  - npm install
+  - yarn install
```

Comment:
We should install without the lockfile on CI since this is an addon. `yarn install --no-lockfile`